assets/cross-sell-carousel.js

```diff
@@ -0,0 +1,33 @@
+class CrossSellCarousel extends HTMLElement {
+  constructor() {
+    super();
+    this.glideInstance = new Glide('[data-glide]', {
```

Comment:
This selector is too generic. You're only other use of glide has a diferent selector, obviously, so this isn't an issue now, but if the page had more and more sliders added, this could cause bugs. Always make your selectors specific as is reasonable.